### ISSUE TITLE
Unciv crash

### ISSUE THREAD

**Issue by mingi8925** [yairm210/Unciv#9338]
**Platform:** Android
**Version:** 4.6.9 (Build 859)
**Rulesets:** [Civ V - Vanilla, Civ V - Gods & Kings]
**Last Screen:** `com.unciv.ui.screens.newgamescreen.NewGameScreen`

--------------------------------

Device Model: SM-N981N
API Level: 30

--------------------------------

**Message:**
```
java.lang.NullPointerException: Attempt to read from field 'float com.badlogic.gdx.math.Rectangle.x' on a null object reference
	at com.unciv.ui.components.NativeBitmapFontData.getGlyph(Fonts.kt:122)
	at com.badlogic.gdx.graphics.g2d.BitmapFont$BitmapFontData.getGlyphs(BitmapFont.java:845)
	at com.unciv.ui.components.NativeBitmapFontData.getGlyphs(Fonts.kt:177)
	at com.badlogic.gdx.graphics.g2d.GlyphLayout.setText(GlyphLayout.java:178)
	at com.badlogic.gdx.graphics.g2d.GlyphLayout.setText(GlyphLayout.java:95)
	at com.badlogic.gdx.scenes.scene2d.ui.Label.computePrefSize(Label.java:162)
	at com.badlogic.gdx.scenes.scene2d.ui.Label.scaleAndComputePrefSize(Label.java:147)
	at com.badlogic.gdx.scenes.scene2d.ui.Label.getPrefWidth(Label.java:244)
	at com.badlogic.gdx.scenes.scene2d.ui.Value$3.get(Value.java:87)
	at com.badlogic.gdx.scenes.scene2d.ui.Table.computeSize(Table.java:827)
	at com.badlogic.gdx.scenes.scene2d.ui.Table.getPrefWidth(Table.java:432)
	at com.badlogic.gdx.scenes.scene2d.ui.Button.getPrefWidth(Button.java:244)
	at com.badlogic.gdx.scenes.scene2d.ui.VerticalGroup.computeSize(VerticalGroup.java:117)
	at com.badlogic.gdx.scenes.scene2d.ui.VerticalGroup.getPrefWidth(VerticalGroup.java:295)
	at com.badlogic.gdx.scenes.scene2d.ui.Value$3.get(Value.java:87)
	at com.badlogic.gdx.scenes.scene2d.ui.Table.computeSize(Table.java:827)
	at com.badlogic.gdx.scenes.scene2d.ui.Table.getPrefWidth(Table.java:432)
	at com.badlogic.gdx.scenes.scene2d.ui.SplitPane.getPrefWidth(SplitPane.java:173)
	at com.badlogic.gdx.scenes.scene2d.ui.Value$3.get(Value.java:87)
	at com.badlogic.gdx.scenes.scene2d.ui.Table.computeSize(Table.java:827)
	at com.badlogic.gdx.scenes.scene2d.ui.Table.layout(Table.java:953)
	at com.unciv.ui.screens.pickerscreens.PickerPane.layout(PickerPane.kt:65)
	at com.badlogic.gdx.scenes.scene2d.ui.WidgetGroup.validate(WidgetGroup.java:113)
	at com.badlogic.gdx.scenes.scene2d.ui.Table.draw(Table.java:108)
	at com.badlogic.gdx.scenes.scene2d.Group.drawChildren(Group.java:111)
	at com.badlogic.gdx.scenes.scene2d.Group.draw(Group.java:58)
	at com.badlogic.gdx.scenes.scene2d.Stage.draw(Stage.java:129)
	at com.unciv.ui.screens.basescreen.UncivStage.access$draw$s80204510(UncivStage.kt:17)
	at com.unciv.ui.screens.basescreen.UncivStage$draw$1.invoke(UncivStage.kt:53)
	at com.unciv.ui.screens.basescreen.UncivStage$draw$1.invoke(UncivStage.kt:53)
	at com.unciv.ui.crashhandling.CrashHandlingExtensionsKt$wrapCrashHandling$1.invoke(CrashHandlingExtensions.kt:17)
	at com.unciv.ui.crashhandling.CrashHandlingExtensionsKt$wrapCrashHandlingUnit$1.invoke(CrashHandlingExtensions.kt:33)
	at com.unciv.ui.crashhandling.CrashHandlingExtensionsKt$wrapCrashHandlingUnit$1.invoke(CrashHandlingExtensions.kt:33)
	at com.unciv.ui.screens.basescreen.UncivStage.draw(UncivStage.kt:53)
	at com.unciv.ui.screens.basescreen.BaseScreen.render(BaseScreen.kt:83)
	at com.badlogic.gdx.Game.render(Game.java:48)
	at com.unciv.UncivGame.access$render$s2211858(UncivGame.kt:125)
	at com.unciv.UncivGame$wrappedCrashHandlingRender$1.invoke(UncivGame.kt:143)
	at com.unciv.UncivGame$wrappedCrashHandlingRender$1.invoke(UncivGame.kt:143)
	at com.unciv.ui.crashhandling.CrashHandlingExtensionsKt$wrapCrashHandling$1.invoke(CrashHandlingExtensions.kt:17)
	at com.unciv.ui.crashhandling.CrashHandlingExtensionsKt$wrapCrashHandlingUnit$1.invoke(CrashHandlingExtensions.kt:33)
	at com.unciv.ui.crashhandling.CrashHandlingExtensionsKt$wrapCrashHandlingUnit$1.invoke(CrashHandlingExtensions.kt:33)
	at com.unciv.UncivGame.render(UncivGame.kt:456)
	at com.badlogic.gdx.backends.android.AndroidGraphics.onDrawFrame(AndroidGraphics.java:474)
	at android.opengl.GLSurfaceView$GLThread.guardedRun(GLSurfaceView.java:1591)
	at android.opengl.GLSurfaceView$GLThread.run(GLSurfaceView.java:1286)

```

**Comment by yairm210** [comment]
@SomeTroglodyte this is not an isolated incident, I got another report on this from a modless game.
In order to stop the crashes, I'm planning on releasing a patch reverting the font disposal, and then we can figure out what happened here and how we can solve in our own time

**Comment by yairm210** [comment]
Should be resolved with 32041d5b
